frappe/model/base_document.py

```diff
@@ -921,13 +921,25 @@ def _validate_selects(self):
 				)
 
 	def _validate_data_fields(self):
-		# data_field options defined in frappe.model.data_field_options
+		from frappe.utils import (
+			split_emails,
+			validate_email_address,
+			validate_name,
+			validate_phone_number,
+			validate_phone_number_with_country_code,
+			validate_url,
+		)
+
 		for phone_field in self.meta.get_phone_fields():
 			phone = self.get(phone_field.fieldname)
-			frappe.utils.validate_phone_number_with_country_code(phone, phone_field.fieldname)
+			validate_phone_number_with_country_code(phone, _(phone_field.fieldname))
```

Comment:
The translation should happen inside `validate_phone_number_with_country_code` since that's where all translation is handled - especially given the argument name says "fieldname"

Reply:
The parameter is badly named, but changing it would be breaking. And `_(fieldname)` also doesn't look right, since there are no translations for fieldnames. 🤷‍♂️

Reply:
I've removed the offending commit for now.